src/promptflow-core/promptflow/executor/_script_executor.py

```diff
@@ -33,7 +33,7 @@
 class ScriptExecutor(FlowExecutor):
     def __init__(
         self,
-        flow_file: Path,
+        flow_file: Union[Path, str, Callable],
```

Comment:
it looks a little confusing if we still name it `flow_file` here

Reply:
Let's use other PRs for the renaming.